app/networkapi/templates/admin/pages/page/change_list.html

```diff
@@ -24,15 +27,13 @@
 
 {% block content %}
 <div id="content-main">
-    {% if has_add_permission %}
```

Comment:
this file is no longer used, the `<a>` that it generates just straight up lives in the `admin.html` file now